Gemfile

```diff
@@ -109,6 +109,7 @@ end
 
 group :test do
   gem "capybara"
+  gem "capybara-screenshot"
```

Comment:
I think `capybara-screenshot` is included transitively.

Reply:
I don't believe it was in the `Gemfile.lock` prior to the changes in this branch. I can try commenting this out, re-bundling, and seeing whether the `require` still works though?

Reply:
The `require` fails without `capybara-screenshot` in the Gemfile, alas.